packages/remix-dev/__tests__/migrations/convert-to-javascript-test.ts

```diff
@@ -68,56 +103,53 @@ const makeApp = () => {
   return projectDir;
 };
 
-const runConvertToJavaScriptMigration = (projectDir: string) =>
-  run([
-    "migrate",
-    "--migration",
-    "convert-to-javascript",
-    projectDir,
-    "--force",
-  ]);
+const getRunArgs = (projectDir: string) => [
+  "migrate",
+  "--migration",
+  "convert-to-javascript",
+  projectDir,
+  "--force",
+];
+const runConvertToJavaScriptMigrationProgrammatically = (projectDir: string) =>
+  run(getRunArgs(projectDir));
+const runConvertToJavaScriptMigrationViaCLI = (projectDir: string) =>
+  spawnSync(
+    "node",
+    [
+      "--require",
+      require.resolve("esbuild-register"),
+      "--require",
+      join(__dirname, "..", "msw.ts"),
+      resolve(__dirname, "..", "..", "cli.ts"),
+      ...getRunArgs(projectDir),
+    ],
+    { cwd: projectDir }
+  ).stdout?.toString("utf-8");
 
 describe("`convert-to-javascript` migration", () => {
-  it("runs successfully", async () => {
+  it("runs successfully when ran via CLI", async () => {
     let projectDir = makeApp();
-    let config = await readConfig(projectDir);
 
-    await runConvertToJavaScriptMigration(projectDir);
+    let output = runConvertToJavaScriptMigrationViaCLI(projectDir);
 
-    let jsConfigJson: TsConfigJson = readJSONSync(
-      join(projectDir, "jsconfig.json")
-    );
-    expect(jsConfigJson.include).not.toContain("remix.env.d.ts");
-    expect(jsConfigJson.include).not.toContain("**/*.ts");
-    expect(jsConfigJson.include).toContain("**/*.js");
-    expect(jsConfigJson.include).not.toContain("**/*.tsx");
-    expect(jsConfigJson.include).toContain("**/*.jsx");
-
-    let packageJson: PackageJson = readJSONSync(
-      join(projectDir, "package.json")
-    );
-    expect(packageJson.devDependencies).not.toContain("@types/react");
-    expect(packageJson.devDependencies).not.toContain("@types/react-dom");
-    expect(packageJson.devDependencies).not.toContain("typescript");
-    expect(packageJson.scripts).not.toContain("typecheck");
+    await checkMigrationRanSuccessfully(projectDir);
+
+    console.log("debug: ---START---");
+    console.log("debug: output", output);
+    console.log("debug: ---END---");
```

Comment:
I think we want to remove these

Reply:
Indeed.
Just left them here for @pcattori to be able to have a look at what's going wrong with what he proposed in https://github.com/remix-run/remix/pull/3529#discussion_r902850714

Reply:
@MichaelDeBoey : we can remove these debugging statements now, right?